app/javascript/dashboard/routes/dashboard/settings/labels/AddLabel.vue

```diff
@@ -94,20 +98,24 @@ export default {
       return color;
     },
     addLabel() {
-      this.$store
-        .dispatch('labels/create', {
-          color: this.color,
-          description: this.description,
-          title: this.title,
-          show_on_sidebar: this.showOnSidebar,
-        })
-        .then(() => {
-          this.showAlert(this.$t('LABEL_MGMT.ADD.API.SUCCESS_MESSAGE'));
-          this.onClose();
-        })
-        .catch(() => {
-          this.showAlert(this.$t('LABEL_MGMT.ADD.API.ERROR_MESSAGE'));
-        });
+      if (this.hasSameLabel) {
+        this.showAlert(this.$t('LABEL_MGMT.ADD.API.NAME_EXISTS'));
+      } else {
+        this.$store
+          .dispatch('labels/create', {
+            color: this.color,
+            description: this.description,
+            title: this.title,
+            show_on_sidebar: this.showOnSidebar,
+          })
+          .then(() => {
+            this.showAlert(this.$t('LABEL_MGMT.ADD.API.SUCCESS_MESSAGE'));
+            this.onClose();
+          })
+          .catch(() => {
+            this.showAlert(this.$t('LABEL_MGMT.ADD.API.ERROR_MESSAGE'));
```

Comment:
Show message if the error object contains any messages, other way show default error message

Reply:
fixes by https://github.com/chatwoot/chatwoot/pull/4161/commits/bfc3efe3fb7d5703eb1612830ec15a3da61c1839